recipes/h2/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "h2" %}
+{% set version = "3.0.1" %}
+{% set sha256 = "b2962f883fa392a23cbfcc4ad03c335bcc661be0cf9627657b589f0df2206e64" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Can this be a [`noarch` package](https://conda-forge.org/docs/meta.html#building-noarch-packages)?

Reply:
Looks like not, because [there's a missing version-dependent dep](https://github.com/conda-forge/staged-recipes/pull/4742/files#r159947305).